Userland/Libraries/LibFileSystemAccessClient/Client.cpp

```diff
@@ -131,11 +131,11 @@ void Client::handle_prompt_end(i32 request_id, i32 error, Optional<IPC::File> co
             break;
         [[fallthrough]];
     default:
-        auto maybe_message = ErrorOr<String>({});
-        if (error == ECONNRESET)
-            maybe_message = String::formatted("FileSystemAccessClient: {}", Error::from_errno(error));
-        else
-            maybe_message = String::formatted("{} \"{}\" failed: {}", action, *chosen_file, Error::from_errno(error));
+        auto maybe_message = [&]() -> ErrorOr<String> {
```

Comment:
IMO, the only advantage to use `lambda-initialization` is to allow the object to be `const`
Otherwise, initializing `maybe_message` with an empty `String` is simpler